### PR TITLE
chore(release): pull release/v1.66.0 into main

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3367/test-report.html

**Review by ItsSudip — DISMISSED** [review]

File: src/v1/destinations/emarsys/networkHandler.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    const errorMsg = JSON.stringify(errors['<NO_KEY_ID>']);
+    return { isAbortable: true, errorMsg };
+  }
+
+  // Return false and an empty error message if no error is found
+  return { isAbortable: false, errorMsg: '' };
+}
+
+const responseHandler = (responseParams) => {
+  const { destinationResponse, rudderJobMetadata, destinationRequest } = responseParams;
```
Comment (krishna2020) [inline src/v1/destinations/emarsys/networkHandler.js:45]: @shrouti1507 destType is 3rd parameter, can you recheck the contract updated in native integration.ts ?

Comment (shrouti1507) [reply src/v1/destinations/emarsys/networkHandler.js:45]: Screenshot 2024-05-14 at 3 03 47 PM
@krishna2020 added it here

Comment (krishna2020) [reply src/v1/destinations/emarsys/networkHandler.js:45]: see the contract in nativeIntegration.ts & in networkHandler.js
you are passing destType & referring it as destinationRequest here

Comment (shrouti1507) [reply src/v1/destinations/emarsys/networkHandler.js:45]: did not get you, we are passing response params, which is an object, and in that object we have introduced a key called destinatination Request. And we are destructuring it there. Just for a simple example like below 
Screenshot 2024-05-14 at 4 06 59 PM

File: src/v1/destinations/emarsys/networkHandler.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  const message = `[EMARSYS Response V1 Handler] - Request Processed Successfully`;
```
Comment (krishna2020) [inline src/v1/destinations/emarsys/networkHandler.js:46]: Use destType from responseParams

**Review by krishna2020 — APPROVED** [review]

**Pull request merged**